insights/parsers/losetup.py

```diff
@@ -0,0 +1,82 @@
+"""
+LoSetup - command ``/usr/sbin/losetup -l``
+==========================================
+
+This parser reads the output of ``/usr/sbin/losetup -l`` into a list of entries.
+Each entry is a dictionary of headers:
+
+* ``NAME`` - the path name of the loop back device (strings)
+* ``SIZELIMIT`` - the data end position of backing file in bytes (integer)
+* ``OFFSET`` - the data start position of backing file in bytes (integer)
+* ``AUTOCLEAR`` - the autoclear flag (boolean)
+* ``RO`` - the read only flag (boolean)
+* ``BACK-FILE`` - the path of the backing file (strings)
+* ``DIO`` - the direct I/O flag (boolean)
+* ``LOG-SEC`` - the logical sector size of the loop device in bytes (integer)
+
+Sample output of ``losetup -l`` command is::
+
+    NAME       SIZELIMIT OFFSET AUTOCLEAR RO BACK-FILE      DIO LOG-SEC
+    /dev/loop0         0      0         0  0 /root/disk.img   1     512
+
+Examples:
+
+    >>> type(losetup)
+    <class 'insights.parsers.losetup.LoSetup'>
+    >>> len(losetup)
+    1
+    >>> losetup[0]['NAME']
+    '/dev/loop0'
+    >>> losetup[0]['RO']
+    False
+    >>> losetup[0]['DIO']
+    True
+    >>> losetup[0]['NAME']
+    '/dev/loop0'
+    >>> losetup[0]['SIZELIMIT']
+    0
+    >>> losetup[0]['OFFSET']
+    0
+    >>> losetup[0]['AUTOCLEAR']
+    False
+    >>> losetup[0]['RO']
+    False
+    >>> losetup[0]['BACK-FILE']
+    '/root/disk.img'
+    >>> losetup [0]['DIO']
+    True
+    >>> losetup[0]['LOG-SEC']
+    512
+"""
+from insights import CommandParser, parser
+from insights.specs import Specs
+from insights.parsers import parse_delimited_table
+
+
+@parser(Specs.losetup)
+class LoSetup(CommandParser):
```

Comment:
@takayuki-nagata Could you switch the class to inheriting a list instead of using the older self.data variable format? You can look at the ChronycSources parser in the ntp_sources.py file as an example. Basically just inherit list and do self.extend(parser_delimited_table(content)).

Reply:
I have switched the class to inheriting a list, and have changed to storing a result of parser_delimited_table() to itself. In addition, I have added data() as a property.